app/serializers/activitypub/note_serializer.rb

```diff
@@ -204,6 +206,22 @@ def icon
     def thumbnail?
       object.thumbnail.present?
     end
+
+    def width?
+      object.file.meta.is_a?(Hash) && object.file.meta.dig('original', 'width').is_a?(Integer)
```

Comment:
I think this can be simplified into `object.file.meta&.dig('original', 'width').present?`

Reply:
Indeed, I've updated. :slightly_smiling_face: